codec_context.go

```diff
@@ -341,6 +346,16 @@ func (cc *CodecContext) SetHardwareDeviceContext(hdc *HardwareDeviceContext) {
 	}
 }
 
+func (cc *CodecContext) SetHardwareFrameContext(hfc *HardwareFrameContext) {
+	if cc.c.hw_frames_ctx != nil {
```

Comment:
Can you use `cc.hfc.c` instead of `cc.c.hw_frames_ctx` here and the line below as well?